ui/compose.lua

```diff
@@ -1,6 +1,7 @@
 local FORMNAME = "mail:compose"
+local msg_id = nil
```

Comment:
There's probably big issues with this. Think what happens when 2 different players opens formspec and selects `msg_id`.

How is this scenario handled here?